### ISSUE TITLE
[API] Pass through all openai api parameters to models

### ISSUE THREAD

**Issue by philwinder** [helixml/helix#337]
We're not passing through (at least):

- top_p/top_k
- temperature
- max_tokens
- seed
- logit_probs
- etc

**Comment by lukemarsden** [comment]
Yeah, function calling is a big one here. Need to pass those params back and forth

**Comment by lukemarsden** [comment]
this is done now i think
